src/test/java/unit/CloneWorkspace.java

```diff
@@ -55,6 +56,8 @@ public class CloneWorkspace extends ClearContextUnit {
   private UFWorkspace sourceWorkspace;
   private UFWorkspace destinationWorkspace;
 
+  private static final Random RANDOM = new Random();
```

Comment:
Delete, it's no longer used now that you're calling TestUtils.appendRandomNumber()